index.html

```diff
@@ -1498,6 +1515,53 @@
+design their products in ways that make it a lot easier for [=people=] to disclose information than not, whereas
+[=people=] typically have to manually wade through options, repeated prompts, and deceptive patterns. In many
+cases, the absence of data — when a [=person=] refuses to provide some information — can also be identifying
+or revealing. Additionally, APIs can be defined or implemented in rigid ways that can prevent people from
+accessing useful functionality (eg. if my geolocation is forcefully set to match my GPS, a restaurant-finding
+site might restrict me to my current location when I want to look for restaurants in a city I will be visiting
+this weekend).
+
+[=User agents=] should make it simple for [=people=] to present the identity they wish to and to provide information
+about themselves or their devices in ways that they control. This empowers [=people=] to live in obscurity
+([[?Lost-In-Crowd]], [[?Obscurity-By-Design]]) by rendering the provision of information according to their own
+preferences as easy as possible ([[?Obfuscation]]).
+
+APIs should be designed such that they do not expect to provide the truth about a [=person=] as provided for
+instance by a sensor or recorded data. Instead, the API could indicate a [=person=]'s preference, a [=person=]'s
+chosen identity, a [=person=]'s query or interest, or a [=person=]'s selected communication style. Directing APIs
+to respond with information that may differ from the truth can be essential in protecting [=people=], is a key
+component of fingerprinting mitigation, and can be required for auditing purposes such that an auditor may
+pretend to be someone else in order for instance to detect discrimination.
+
+In some cases, sites do not abide by data minimisation principles and request more information than they realistically
+require. This principle supports [=people=] in defending under such cases.
+
+It is worth noting that this principle deliberately empowers [=people=] to provide deceptive information. In the
+general case, the ability for people to present as they wish is more important than the ability of computers to
+prevent them from doing so. Sites should include deception in their threat modelling.
+
+Examples of ways in which a [=user agent=] may support this principle include:
+* Generating domain-specific email addresses or other directed identifiers so that [=people=] can log into the
+  site without becoming recognisable across contexts.
+* Offering the option to produce fake geolocation and accelerometry data, with parameters specified by the [=user=].
+* Making it easy to upload a picture while pretending to have it taken by the camera.
+* Faking the acceptance of notification prompts (or pretending that notifications have already been accepted).
+* Filtering outbound network traffic so as to replace specific strings (TCF, hashed email, navigation identifiers).
+* Randomly swapping third-party cookies with the browsers of other [=people=].
```

Comment:
I'm not sure this is a helpful example for the reader. 

Third-party cookies may not always be widely supported. Swapping of cookies could be an obfuscation method, but could also lead to security vulnerabilities.